tests/src/test/scala/org/apache/openwhisk/core/cli/test/WskCliBasicUsageTests.scala

```diff
@@ -653,37 +653,43 @@ class WskCliBasicUsageTests extends TestHelpers with WskTestHelpers {
     val createAnnots = Map(createKey -> createValue, origKey -> origValue)
     val updateAnnots =
       Map(updateKey -> updateValue, origKey -> overwrittenValue)
+    val baseAnnotations =
+      Parameters("web-export", JsTrue) ++
+        Parameters("raw-http", JsFalse) ++
+        Parameters("final", JsTrue) ++
+        Parameters("exec", "nodejs:6")
+    val createAnnotations = if (requireAPIKeyAnnotation) {
+      baseAnnotations ++
+        Parameters(WhiskAction.provideApiKeyAnnotationName, JsFalse) ++
+        Parameters(createKey, createValue) ++
+        Parameters(origKey, origValue)
+    } else {
+      baseAnnotations ++
+        Parameters(createKey, createValue) ++
```

Comment:
@dubee - should the `createKey` and `origKey` be included in the base annotation list?

Reply:
`createKey` and `origKey` get overrwritten with an action update, so I did not include those in `baseAnnotations`.